### ISSUE TITLE
Dune 2.0.0b failing with 4.05.0+mingw32c

### ISSUE THREAD

**Issue by yallop** [fdopen/opam-repository-mingw#64]
The `dune 2.0.0b` build is failing with `4.05.0+mingw32c`.  The following is from the [Appveyor build logs for ctypes](https://ci.appveyor.com/project/yallop/ocaml-ctypes/builds/29372901/job/c3im24d0khtv3vxt):

```
#=== ERROR while compiling dune.2.0.0b ========================================#
# context              2.0.5 | win32/x86_64 | ocaml-variants.4.05.0+mingw32c | git+https://github.com/fdopen/opam-repository-mingw.git#opam2
# path                 ~/.opam/ocaml-variants.4.05.0+mingw32c/.opam-switch/build/dune.2.0.0b
# command              C:\cygwin64\bin\sh.exe build.sh
# exit-code            1
# env-file             ~/.opam/log/dune-1996-126656.env
# output-file          ~/.opam/log/dune-1996-126656.out
### output ###
# mv: cannot stat '=./dune.32.exe': No such file or directory
```

**Comment by fdopen** [comment]
fixed by 80dfc4950759ab4a1351dce3974eba126ba37bee

**Comment by yallop** [comment]
Thanks for the quick fix!  I can confirm that this is working again in the [ctypes Appveyor build](https://ci.appveyor.com/project/yallop/ocaml-ctypes/branch/master/job/9if411bb567x2773).
